pom.xml

```diff
@@ -24,7 +24,7 @@
     <version>2.0.1-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.319</jenkins.version>
+        <jenkins.version>2.319.1</jenkins.version>
```

Comment:
Is this version change necessary for this feature?

Reply:
This is just to run in local. I will revert it.